blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -796,9 +808,8 @@ public void testCopyBlobReplaceMetadata() throws Exception {
          } finally {
             Closeables2.closeQuietly(is);
          }
-         // TODO: S3 overrideMetadataWith also overrides system metadata
-         // TODO: Swift does not preserve system metadata
-         //checkContentMetadata(toBlob);
+         // TODO: Swift copy object *appends* user metadata, does not overwrite
```

Comment:
I don't understand how to do this fixup with the current API -- `ObjectApi.updateMetadata` writes user metadata but overwrites content metadata and `ObjectApi.updateRawMetadata` has some strange semantics.